string queryFile
---
bwi_kr/AnswerSetMsg answerSet